cd/auto-generate/README.md

```diff
@@ -0,0 +1,28 @@
+### Introduction
+
+This directory contains `auto-generate-controllers.sh` script along with template
+files that provide the content for creating github issues and pull requests using
+ack-bot.
+
+
+### How to add new AWS service for controller auto generation
+To enable new services for auto generating controllers on new ACK code-generator
+release,
+
+1. Create a custom label named `ack-bot-codegen` on `aws-controllers-k8s/$SERVICE-controller`
```

Comment:
we could sync labels using prow labels plugin as well

Reply:
I did check prow's label plugin. These 3 labels are supported out of the box ' "kind/*", "priority/*" and "area/*" '
I have not worked with additional-label configuration yet, but If i can successfully add 'ack-bot-codegen' label into prow configuration, I will remove this step.
But I will do it as part of another PR.